spec/requests/atmosphere/api/clew_spec.rb

```diff
@@ -172,20 +167,29 @@ def cs
         expect(clew_at_response['appliance_types'][1]['matched_flavor']).to clew_flavor_eq  flavor
 
         #TODO complete this test to check all the returned fields
-
       end
+    end
+
+    it 'does not return AT with visible_to owner when user is not owner' do
+      user = create(:user)
+      cs = create(:compute_site)
+      create(:flavor, compute_site: cs)
+
+      owned_at = create(:appliance_type, visible_to: :owner, author: user)
+      create(:virtual_machine_template,
+             compute_site: cs, appliance_type: owned_at)
+      not_owned_at = create(:appliance_type, visible_to: :owner)
+      create(:virtual_machine_template,
+             compute_site: cs, appliance_type: not_owned_at)
 
+      get api("/clew/appliance_types", user)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.